geomesa-features/geomesa-feature-avro/src/main/scala/org/locationtech/geomesa/features/avro/AvroSimpleFeatureUtils.scala

```diff
@@ -321,6 +328,8 @@ object AvroSimpleFeatureUtils {
       case "float"   => (4, (bb, v) => bb.putFloat(v.asInstanceOf[Float]))
       case "date"    => (8, (bb, v) => bb.putLong(v.asInstanceOf[Date].getTime))
       case "boolean" => (1, (bb, v) => if (v.asInstanceOf[Boolean]) bb.put(1.toByte) else bb.put(0.toByte))
+      case "uuid"    => (16, (bb, v) => putUUID(bb, v.asInstanceOf[UUID]))
+      case "byte[]"  => (-1, (bb, v) => putBytes(bb, v.asInstanceOf[Array[Byte]]))
```

Comment:
i noticed in simple feature types it accepts `byte[]` or `Bytes` - does that affect this line at all?